routes/settings-routes.js

```diff
@@ -5,12 +5,13 @@ const utils = require('../utils');
 const rbac = require('../rbac');
 logger.module = 'settings-routes';
 
-
 const authCheck = (req, res, next) => {
-  if(!req.user){
+  if(!req.user && process.env.TESTING == undefined){
       // if not logged in
+      console.log('bouncing');
```

Comment:
change to logger, unless this is before the logger module

Reply:
I just removed them as they're no longer in master.